### PR TITLE
Use min version once version api unreachable

### PR DESCRIPTION
**Pull request by mustard-mh**
## Related Issue(s)

Fixes #

## How to test

Change gitpod host to incorrect one, see if it failed with version `0.0.0`

### CODE REVIEW

File: src/featureSupport.ts [hunk 1]
```diff
@@ -32,39 +37,59 @@ export class GitpodVersion {
...
+            } else {
+                return undefined;
+            }
         }
+    };
 
-        gitpodRawVersion = await versionResponse.text();
-    } catch (e) {
-        return undefined;
+    const gitpodRawVersion = await fetchVersion();
```
Comment (jeanp413) [inline src/featureSupport.ts:72]: if `gitpodRawVersion` is undefined then fallback to GitpodVersion.Min

File: src/featureSupport.ts [hunk 2]
```diff
@@ -32,39 +37,59 @@ export class GitpodVersion {
...
+    if (!gitpodRawVersion) {
+        return {
```
Comment (jeanp413) [inline src/featureSupport.ts:74]: Maybe add one more log here telling we are using min version as default

**Review by jeanp413 — APPROVED** [review]

File: src/featureSupport.ts (hunk 2 above)
Comment (mustard-mh) [reply src/featureSupport.ts:76]: Added

**Comment by mustard-mh** [comment]
It's not correct!! @jeanp413 , the static string have no 'release-' prefix 😱

image

**Comment by mustard-mh** [comment]
I'm going to push directly to fix it, no we publish a lot of versions

**Comment by mustard-mh** [comment]
No, my test cases are wrong, we have code below, if add `release-` prefix, it's not correct because it will remove last number
```
        if (semver.valid(gitpodVersion)) {
            this.version = gitpodVersion;
        }
```

**Comment by mustard-mh** [comment]
I'm going to create a unit test PR for it 🥲

**Pull request merged**